src/custom/components/NetworkAlert/NetworkAlert.tsx

```diff
@@ -0,0 +1,184 @@
+// WARNING: THIS IS A MOD!!!
```

Comment:
why do we need this comment

Reply:
so dangerous 😨

Reply:
Because the file name doesn't have Mod in it due to how the original file is setup.
It's not exported as `src/components/NetworkAlert.tsx` which we would mod as `src/custom/components/NetworkAlert/index.tsx`
Instead it has the structure `src/components/NetworkAlert/NetworkAlert.tsx`

So, added the comment just so we know it's not a original custom file we created.

Reply:
But then the mod is still possible, right, just needs to be `src/components/NetworkAlert/NetworkAlert/index.tsx`

Reply:
Oh shit, didn't consider that 🤦 

I'll give that a try

Reply:
No more warnings \o/